tmt/steps/provision/mrack.py

```diff
@@ -40,6 +42,7 @@
 
 # Type annotation for "data" package describing a guest instance. Passed
 # between load() and save() calls
+MRACK_CONFIG_FILE = os.environ.get('MRACK_CONFIG_FILE') or 'mrack-hw-filters.yaml'
```

Comment:
1. comment is now separated from the structures it's commenting by this addition, I suppose this particular line should move before the command.
2. `os.environ.get()` supports default value, i.e. `os.environ.get('MRACK_CONFIG_FILE', 'mrack-hw-filters.yaml')` would work as designed.
3. the environment variable says `CONFIG_FILE` which is not exactly true - it's not a config file for mrack, just for one of the steps. And the default filename proves that, it's not `mrack-config.yaml`, but much better `mrack-hw-filters.yaml`. So how about `MRACK_HW_FILTERS_CONFIG_FILE` or something similar?
4. all environment variables accepted by tmt should have `TMT_` prefix, see https://tmt.readthedocs.io/en/stable/overview.html#variables. We cannot let tmt to conflict with other apps.

Reply:
It would be nice to mention the config file and its structure in the plugin help, i.e. in its docstring, with an example or two. It does not have to do much now, but it would help other developers to understand what and why - I know it, because it comes from Artemis, but others might be lost during reviews. And then there are users :)